person.rb

```diff
@@ -1,19 +1,20 @@
 require './decorator'
 
 class Person < Nameable
+  # accesor get and set method
+  attr_accessor :name, :age
+  # accesor get method
+  attr_reader :id, :rentals
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
```

Comment:
- [ ] Hi. as part of the requirement, you are supposed to create a has-many relationship side of Person and Rental (a person has many rentals). So, you are supposed to create a method for adding rentals in the person class and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
Thank you @m05e5 , let me add that too!